appveyor.yml

```diff
@@ -0,0 +1,12 @@
+version: 0.1.7.{build}
+
+image:
+- Visual Studio 2015
+- Visual Studio 2013
+
+
+build_script:
+- cmake --build . --clean-first
```

Comment:
This statement runs a build, but as far as I remember it does not generate the project files, makefiles, etc. Above this line, try adding:

    - cmake .

This is what I did to generate the VS project files on my windows box.